hadoop-hdds/interface-server/src/main/proto/ScmServerDatanodeHeartbeatProtocol.proto

```diff
@@ -499,6 +499,8 @@ message DiskBalancerReportProto {
   required bool isRunning = 1;
   optional uint64 balancedBytes = 2;
   optional DiskBalancerConfigurationProto diskBalancerConf = 3;
+  optional int64 successCount = 4;
```

Comment:
Please use uint64, instead of int64.

Reply:
successCount -> successMoveCount
failureCount -> failureMoveCount

Please make sure the naming is consistent cross multiple data structure.  For example, you either name both as sucessCount DiskBalancerReportProto and  DatanodeDiskBalancerInfoProto, or both as successMoveCount.